pkg/collector/collector.go

```diff
@@ -20,6 +25,10 @@ const (
 	NetworkOutbound
 	// NodeLogs defines NodeLogs Collector Type
 	NodeLogs
+	// Osm defines Osm Collector Type
```

Comment:
💡 Will recommend to add what OSM and SMI stand for in comments will be a good place.